float32 left_drive_delta
float32 right_drive_delta
float32 time_delta